pyaedt/generic/DataHandlers.py

```diff
@@ -482,6 +486,33 @@ def to_aedt(code):
     return return_code
 
 
+def str_to_bool(s):
+    """Convert a string `"True"` or `"False"` to its corresponding Boolean value.
+
+    If the passed arguments are not relevant in the context of conversion, the argument
+    itself is returned.  This method can be called using the `map()` function to
+    ensure conversion of Boolean strings in a list.
+
+    Parameters
+    ----------
+    s: str
```

Comment:
Description? Even if it is as simple as the ``"True"`` or ``"False"`` string to convert.

Reply:
added description